docs/pages/versions/unversioned/sdk/tracking-transparency.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Tracking Transparency
+sourceCodeUrl: 'https://github.com/expo/expo/tree/master/packages/expo-tracking-transparency'
+---
+
+import APISection from '~/components/plugins/APISection';
+import InstallSection from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+import SnackInline from '~/components/plugins/SnackInline';
+
+A library for requesting permission to track the user or their device. Examples of data used for tracking include email address, device ID, advertising ID, etc. This permission is only necessary on iOS 14.5 and higher.
+
+For more information on Apple's new App Tracking Transparency framework, please refer to their [documentation](https://developer.apple.com/app-store/user-privacy-and-data-use/).
+
+<PlatformsSection android emulator ios simulator web />
+
+## Installation
+
+<InstallSection packageName="expo-tracking-transparency" />
+
+## Usage
+
+<SnackInline label='Basic tracking transparency usage' dependencies={['expo-tracking-transparency']}>
```

Comment:
we may want to remove snackinline from the sdk 41 version of this docs page when we cut it for the patch release, because the patch release may not go out to snack at the same time